synapse-module/default.nix

```
@@ -394,5 +396,19 @@ in
         };
       };
     };
+
+    services.matrix-synapse-next.settings.extra_well_known_client_content."org.matrix.msc3575.proxy" = mkIf cfg.enableSlidingSync {
+      url = "https://${config.services.matrix-synapse.sliding-sync.publicBaseUrl}";
+    };
+    services.matrix-synapse.sliding-sync = mkIf cfg.enableSlidingSync {
+      enable = true;
+      enableNginx = cfg.enableNginx;
```

Is this option even needed if we're just going to blatantly set it equal to the main module option?

```suggestion
      enableNginx = lib.mkDefault cfg.enableNginx;
```

---

Yeah, the sliding sync module is completely disjoint from the synapse module, and could be exposed by its own